modules/feature-upload.php

```diff
@@ -291,7 +304,7 @@
 				exit();
 			}
 			
-			$filepath = $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
+			$filepath = getcwd() . "/" . $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
```

Comment:
Hrm... the more I mess with this, the more I think that the problem is different than I had thought.  When I use the original code for the filepath (without the `getcwd`), but put it through `realpath`, it has the correct value.  I found that some people had problems with imagick not working with relative paths, but all the bug reports I can find are related to PHP 5.4, not 7.  I assume that the original works fine for you?